spec/outputs/http_spec.rb

```diff
@@ -520,22 +415,28 @@ def start_app_and_wait(app, opts = {})
   end
 end
 
-describe LogStash::Outputs::Http do # different block as we're starting web server with TLS
+RSpec.describe LogStash::Outputs::Http do # different block as we're starting web server with TLS
 
   @@default_server_settings = TestApp.server_settings.dup
 
   before do
-    cert, key = WEBrick::Utils.create_self_signed_cert 2048, [["CN", ssl_cert_host]], "Logstash testing"
-    TestApp.server_settings = @@default_server_settings.merge({
-       :SSLEnable       => true,
-       :SSLVerifyClient => OpenSSL::SSL::VERIFY_NONE,
-       :SSLCertificate  => cert,
-       :SSLPrivateKey   => key
-    })
+    TestApp.server_settings = @@default_server_settings.merge(webrick_config)
 
     TestApp.last_request = nil
 
-    @server = start_app_and_wait(TestApp)
+    @server = start_app_and_wait(TestApp, &server_setup)
+  end
+
+  let(:server_setup) { lambda { |_| } }
```

Comment:
why this lambda is required?

Reply:
actually isn't required - thought I could use the hook to force the `context. ssl_version=`
but turns out it's too late and needs a patch: https://github.com/logstash-plugins/logstash-output-http/pull/131/files#diff-89eebfcbc0f14b6d989517837ca1e94fce4e2ce9a03233641cd936f2b8d2ed94R26

Reply:
got it. You can consider removing lambda and start_app_and_wait(TestApp, ~~&server_setup~~)